Sources/SwiftProtobuf/Google_Protobuf_Any+Registry.swift

```diff
@@ -32,7 +32,7 @@ internal func typeName(fromMessage message: Message) -> String {
   return messageType.protoMessageName
 }
 
-internal let defaultTypePrefix: String = "type.googleapis.com"
+public let defaultAnyTypeURLPrefix: String = "type.googleapis.com"
```

Comment:
I could back this out for now.  (Or maybe keep the name change but leave the symbol `internal` for now?)  As I mentioned earlier, Swift 4 isn't final, so this may not be really needed.